db/fakeTheData.js

```diff
@@ -1,33 +1,39 @@
 const faker = require('faker');
 const mongoose = require('mongoose');
 
+const env = require('../server/environment');
 const schema = require('./schema.js');
 
 const dbName = 'luke-chat';
-const dbURL = 'mongodb://localhost';
-const dbPort = 3012;
+const dbURL = `mongodb://${env.dbHost}`;
 
-mongoose.connect(`${dbURL}:${dbPort}/${dbName}`, {
+mongoose.connect(`${dbURL}:${env.dbPort}`, {
   useNewUrlParser: true,
   useUnifiedTopology: true,
+  dbName,
+  user: env.dbUser,
+  pass: env.dbPwd,
 })
   .then(() => {
     // console.log('connected to DB!');
 
     const Chat = mongoose.model('Chat', schema);
 
     const arr = [];
-    for (let i = 0; i < 250; i += 1) {
-      arr.push({
-        userName: faker.internet.userName(),
-        displayName: faker.internet.userName(),
-        displayNameColor: faker.internet.color(),
-        badge: faker.internet.avatar(),
-        badgeText: `${faker.lorem.word()} ${faker.lorem.word()}`,
-        message: faker.lorem.sentence(),
-        videoId: Math.floor(Math.random() * 10) + 1,
-        timeStamp: Math.random() * 10,
-      });
+    for (let videoId = 1; videoId < 100; videoId += 1) {
```

Comment:
Wouldn't this only create 99 videos since videoId starts at 1 and ends at 100?

Reply:
Yes, thanks for catching that!